ios/Runner/Info.plist

```diff
@@ -77,5 +77,7 @@
     </array>
 	<key>UIApplicationSupportsIndirectInputEvents</key>
 	<true/>
+	<key>NSMotionUsageDescription</key>
+	<string>This app requires access to motion sensors to detect phone movements, such as shaking, for enhanced functionality.</string>
```

Comment:
The description needs to be more specific otherwise it might get rejected in the appstore review.